### PR TITLE
fix bit_and bug

### PR DESCRIPTION
**Pull request by JackTan25**
**What type of PR is this?**

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

issue #

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

**Review by nnsgmsone — APPROVED** [review]

**Comment by nnsgmsone** [comment]
The following case result is incorrect
```
cases/function/func_aggr_avg.test][row:180][select a,count(b), sum(b), avg(b), std(b), min(b), max(b), bit_and(b), bit_or(b) from t1 group by a;] was executed failed
2022-07-28 19:45:46 ERROR Executor:159 - [EXPECT RESULT]:
a   count(b)    sum(b)  avg(b)  std(b)  min(b)  max(b)  bit_and(b)  bit_or(b)
1   0   null    null    null    null    null    null    null
2   1   1   1.0 0.0 1   1   0   1
3   1   1   1.0 0.0 1   1   1   1
2022-07-28 19:45:46 ERROR Executor:160 - [ACTUAL RESULT]:
a   count(b)    sum(b)  avg(b)  std(b)  min(b)  max(b)  bit_and(b)  bit_or(b)
1   0   null    null    null    null    null    null    null
2   1   1   1.0 0.0 1   1   0   1
3   1   1   1.0 0.0 1   1   0   1
```

**Review by nnsgmsone — CHANGES_REQUESTED** [review]
The following case result is incorrect:
```
cases/function/func_aggr_avg.test][row:180][select a,count(b), sum(b), avg(b), std(b), min(b), max(b), bit_and(b), bit_or(b) from t1 group by a;] was executed failed
2022-07-28 19:45:46 ERROR Executor:159 - [EXPECT RESULT]:
a   count(b)    sum(b)  avg(b)  std(b)  min(b)  max(b)  bit_and(b)  bit_or(b)
1   0   null    null    null    null    null    null    null
2   1    1      1.0    0.0     1   1   1   1
3   1    1      1.0    0.0   1   1   1   1
2022-07-28 19:45:46 ERROR Executor:160 - [ACTUAL RESULT]:
a   count(b)    sum(b)  avg(b)  std(b)  min(b)  max(b)  bit_and(b)  bit_or(b)
1   0   null    null    null    null    null    null    null
2   1   1   1.0 0.0 1   1   0   1
3   1   1   1.0 0.0 1   1   0   1
```
```
cases/function/func_aggr_avg.test][row:248][select grp, sum(a),count(a),avg(a),std(a),variance(a),bit_or(a),bit_and(a),min(a),max(a),min(c),max(c) from t1 group by grp;] was executed failed
2022-07-28 19:45:46 ERROR Executor:159 - [EXPECT RESULT]:
grp sum(a)  count(a)    avg(a)  std(a)  variance(a) bit_or(a)   bit_and(a)  min(a)  max(a)  min(c)  max(c)
1   1   1   1.0000  0.0                 0.0                 1   1   1   1   a   a
2   5   2   2.5000  0.5                 0.25                3   2   2   3   b   c
3   15  3   5.0000  0.816496580927726   0.6666666666666666  7   4   4   6   C   E
2022-07-28 19:45:46 ERROR Executor:160 - [ACTUAL RESULT]:
grp sum(a)  count(a)    avg(a)  std(a)  variance(a) bit_or(a)   bit_and(a)  min(a)  max(a)  min(c)  max(c)
1   1   1   1.0     0.0                 0.0                 1   0   1   1   a   a
2   5   2   2.5     0.5                 0.25                3   2   2   3   b   c
3   15  3   5.0 0.8164965809277268  0.6666666666666679      7   4   4   6   C   E
```

File: pkg/sql/colexec/agg/bit_xor/bit_xor.go [hunk 1]
```diff
@@ -23,11 +23,11 @@ func New[T1 types.Ints | types.UInts | types.Floats]() *BitXor[T1] {
 func (bx *BitXor[T1]) Grows(_ int) {
```
Comment (fengttt) [inline pkg/sql/colexec/agg/bit_xor/bit_xor.go:23]: Still troubles me, what do you mean by and/or/xor two floating point number?

Comment (JackTan25) [reply pkg/sql/colexec/agg/bit_xor/bit_xor.go:23]: float will be cast to uint64

**Pull request closed without merging**